lib/db/mysql.js

```diff
@@ -554,6 +554,48 @@ module.exports = function (log, error) {
     return this.readFirstResult(GET_UNLOCK_CODE, [uid])
   }
 
+  // METRICS
+
+  // Select : accounts
+  // Where  : createdAt < $1
+  var COUNT_ACCOUNTS_CREATED_BEFORE = 'call countAccountsCreatedBefore_1(?)'
+
+  MySql.prototype.countAccountsCreatedBefore = function (createdBefore) {
```

Comment:
@rfk, in light of the information from https://github.com/mozilla-services/puppet-config/issues/1489#issuecomment-131029979, I think we can remove the `createdBefore` parameter for this query and the one below. Agree or disagree?